packages/playground/remote/src/lib/playground-mu-plugin/0-playground.php

```diff
@@ -1,4 +1,28 @@
 <?php
+// Define Playground globals
+$playground_scope = get_scope_from_url(get_site_url());
```

Comment:
This code doesn't seem to be used anywhere at the moment, right? Let's delay adding it until it's actually needed.

Reply:
It's not used. I found it useful when writing mu-plugins and blueprint PHP requests, but I can always add it as needed. Removed in [459c028](https://github.com/WordPress/wordpress-playground/pull/1083/commits/459c028a352461e7320a6bfcd0a4eb5f50a942de)